.github/copilot-instructions.md

```diff
@@ -0,0 +1,6 @@
+Inside backend, when running scripts always activate the venv first.
+
+```bash
+cd /Users/julianjabin/Documents/git/myriade
+source venv/bin/activate
+```
```

Comment:
@narchok this file should probably be added to .gitignore
```
source .venv/bin/activate
```
add this instruction to shared agents.md

Reply:
@narchok i removed it but be careful later on.